### ISSUE TITLE
[URGENT] New user account for AMANDA for Christopher P. (NN)

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#15121]
> What application are you using?

AMANDA

> Describe the problem.

New user account for AMANDA

> How soon do you need this?

Urgently — Critical that this is addressed today

> Is there anything else we should know?

My new staff needs access to AMANDA as soon as possible so he can perform his job duties. 

> Requested By
Michael S.

[Attachment](https://api.knack.com/v1/applications/595d00ebd315cc4cb98daff4/download/asset/65957ab307331c0028327530/new_tpw_dts_v7_amanda.docx) (41.06kb)

Request ID: DTS24-111760

**Comment by TracyLinder** [comment]
Hi Michael, 

Regarding the urgent request for a new AMANDA account for Christopher, I want to be sure you know we are working on it however it may not be created today as new user account setup isn't as straight forward as we'd all hope. Nate (copied) will reach out when the account is ready or with any questions. 

Thanks,
Tracy

**Comment by Nadin-Nader** [comment]
@Tracy this has been done on #15034. Closing

**Comment by Nadin-Nader** [comment]
Time with Jenice scheduled for 945AM 02/02/24

**Comment by Nadin-Nader** [comment]
@TracyLinder After speaking with Jenice we found that Christopher's network account was not set up properly. He is not a part of any AMANDA distros in Active Directory. A ticket will need to be put in to the Service Desk for Him to be added to all of the AMANDA Active Directory groups/distros.

**Comment by Nadin-Nader** [comment]
Spoke to Christopher Parks and he said he was able to fix the issue. I informed him that if it happens again its likely that the network access groups that he need for AMANDA are missing from his network profile. If it happens again. he will open a service ticket with CTM or create a request with us and we can deal with CTM. I also informed him that i would go ahead and close the current request he has with us.
